mslib/msui/multiple_flightpath_dockwidget.py

```diff
@@ -103,6 +114,58 @@ def remove(self):
         self.map.ax.figure.canvas.draw()
 
 
+class CustomColorDialog(QtWidgets.QDialog):
```

Comment:
this should be moved to mslib.utils.colordialog and then imported.

When you create that new module use our template header.